app/src/main/res/values-v21/styles.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="AppTheme" parent="Theme.AppCompat.NoActionBar">
```

Comment:
Because in order to change the status bar color I need to use `android:statusBarColor` and that is only supported for API 21 and up.